packages/client/src/tests/schema.graphql

```diff
@@ -1347,9 +1347,7 @@ type Query {
   ): RegistrationCountResult
   fetchMarriageRegistration(id: ID!): MarriageRegistration
   fetchRecordDetailsForVerification(id: String!): RecordDetails
-  locationsByParent(parentId: String, type: String): [Location]
-  locationById(locationId: String): Location
-  hasChildLocation(parentId: String): Location
+  hasChildLocation(parentId: String!): Location
```

Comment:
Is this used by something?

Reply:
Yes, it was used by Performance-page. I think a backend call might not be needed here as frontend knows the location structure. But decided not to investigate further, as refactor would've been needed.